### ISSUE TITLE
Failed to start docker.service: Unit docker.service not found.

### ISSUE THREAD

**Issue by SyedeeSimon** [docker/docker#27108]
**Description**

**Steps to reproduce the issue:**
1.
2.
3.

**Describe the results you received:**

**Describe the results you expected:**

**Additional information you deem important (e.g. issue happens only occasionally):**

**Output of `docker version`:**

```
(paste your output here)
```

**Output of `docker info`:**

```
(paste your output here)
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by thaJeztah** [comment]
if you're on a RPM based install (CentOS, Fedora, RHEL), that can be expected; see the warning at the top of the release notes for 1.12 https://github.com/docker/docker/releases/tag/v1.12.1

you can find instructions there how to resolve this

**Comment by thaJeztah** [comment]
oh, looks like I misread, as it's not about the docker.socket; what platform/distro are you on, and how did you install? We need some more information if you want help

**Comment by SyedeeSimon** [comment]
Hi,

This is great to see you step forward to help, I appreciate it. Regarding
the error I posted, that was my bad. I actually put wrong version in
/etc/apt/sources.list.d/docker.list. I forgot that I did upgrade the OS
from 14.04 to 16.o4 LTS. Now I used correct version and get around the
problem.
This is what I used
deb https://apt.dockerproject.org/repo ubuntu-xenial main
I was alos able to start the hello-world container. Now I will step into
more advance usage.

Regards,

Khan Sabah Syedee
Software Engineer,
Works Applications
Mobile: +6593960925
email: syedee_k@worksap.co.jp
